components/dashboard/src/components/org-icon/styles.css

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+
+@layer components {
+    .bg-gradient-1 {
+        background: linear-gradient(45deg, theme('colors.gitpod-kumquat-dark') 25%, theme('colors.gitpod-kumquat') 75%);
+    }
+    .bg-gradient-2 {
+        background: linear-gradient(45deg, theme('colors.gitpod-kumquat-dark') 25%, theme('colors.pink.800') 75%);
+    }
+    .bg-gradient-3 {
+        background: linear-gradient(45deg, theme('colors.gitpod-kumquat-dark') 25%, theme('colors.gitpod-red') 75%);
+    }
+    .bg-gradient-4 {
+        background: linear-gradient(45deg, theme('colors.gitpod-kumquat-dark') 25%, theme('colors.green.700') 75%);
+    }
+    .bg-gradient-5 {
+        background: linear-gradient(45deg, theme('colors.gitpod-kumquat-dark') 25%, theme('colors.yellow.500') 75%);
+    }
+    .bg-gradient-6 {
+        background: linear-gradient(45deg, theme('colors.gitpod-kumquat-dark') 25%, theme('colors.indigo.700') 75%);
+    }
+    .bg-gradient-7 {
+        background: linear-gradient(45deg, theme('colors.gitpod-kumquat-dark') 25%, theme('colors.teal.500') 75%);
+    }
+    .bg-gradient-8 {
+        background: linear-gradient(45deg, theme('colors.gitpod-kumquat-dark') 25%, theme('colors.sky.900') 75%);
+    }
+    .bg-gradient-9 {
+        background: linear-gradient(45deg, theme('colors.gitpod-kumquat-dark') 25%, theme('colors.rose.300') 75%);
+    }
```

Comment:
**thought**: Going with this set of colors for the first iteration sounds great! Once we upgrade Tailwind (https://github.com/gitpod-io/gitpod/issues/11846) we can fine tune the colors more.